src/styles/components/_tooltip.scss

```diff
@@ -3,14 +3,14 @@
 $component: "c-tooltip";
 
 .#{$component} {
-	background-color: $black;
-	border: 1px solid $black;
+	background-color: $black !important;
+	border: 1px solid $black !important;
 	display: inline-block;
 	max-width: 19rem !important;
-	padding: 0.5rem;
-	color: $white;
+	padding: 0.5rem !important;
+	color: $white !important;
 	opacity: 0;
-	font-size: 1.3rem;
```

Comment:
These styles are being overridden by the avo2-components tooltip which uses the same class: .c-tooltip

A cleaner way of fixing this conflicts is to use a different class in the react-components. eg:
.c-tooltip-component

Then we avoid conflicting css rules alltogether